fmc/test-fw/test-rt/memory.x

```diff
@@ -2,7 +2,7 @@
 
 ROM_ORG   = 0x00000000;
 META_ORG  = ROM_SIZE;
-ICCM_ORG  = 0x40002000;
+ICCM_ORG  = 0x40004000;
 DCCM_ORG  = 0x50000000;
 FHT_ORG   = 0x50002000;
```

Comment:
Would it make more intuitive to split ICCM regions for FMC and RT ?
ICCM_FMC_*
ICCM_RT_*

Reply:
Added comment explaining the start address.